src/System.ServiceModel.Primitives/tests/Channels/BehaviorExtensionTest.cs

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Collections.Generic;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Description;
+using System.ServiceModel.Dispatcher;
+using System.Threading;
+using Infrastructure.Common;
+using Xunit;
+using System.Threading.Tasks;
+
+public static class BehaviorExtensionTest
+{
+    [WcfFact]
+    public static void BehaviorExtension_OperationBehaviorAttribute()
+    {
+        string testMessageBody = "BehaviorExtension_OperationBehaviorAttribute";
+        Message inputMessage = Message.CreateMessage(MessageVersion.Default, action: "Test", body: testMessageBody);
+
+        // *** SETUP *** \\
+
+        // Create a mock interactive channel initializer to observe whether
+        // its methods are invoked.
+        bool beginInitializeCalled = false;
+        TestOperationBehaviorAttribute testOperationBehavior = new TestOperationBehaviorAttribute();
+        testOperationBehavior.ApplyClientBehaviorOverride = (description, clientOperation) =>
```

Comment:
I think we need to talk in person about this. This isn't testing what you think it is.

Reply:
Updated the tests